ui/mirage/config.js

```diff
@@ -4,11 +4,25 @@
  */
 
 import ENV from 'vault/config/environment';
-import handlers from './handlers';
-
 // remember to export handler name from mirage/handlers/index.js file
+import * as handlers from './handlers/index';
+import { discoverEmberDataModels } from 'ember-cli-mirage';
```

Comment:
The rest of these changes are from following the `ember-cli-mirage` upgrade guide: https://www.ember-cli-mirage.com/docs/getting-started/upgrade-guide#3-0-upgrade-guide